MonstieWash/Assets/Scripts/UI/UIManager.cs

```diff
@@ -76,23 +83,41 @@ private void InitialiseClipboard()
     {
         foreach (var pair in m_taskList)
         {
-            var newTaskObject = Instantiate(taskTextPrefab, taskContainer.transform);
-            newTaskObject.name = pair.Key;
+            var newTaskObject = Instantiate(taskTextPrefab, startList.transform);
+            
+            newTaskObject.name = pair.Key.ToString();
 
             var newTaskText = newTaskObject.GetComponent<TextMeshProUGUI>();
             newTaskText.fontSize = fontSize;
-            newTaskText.text = $"{newTaskObject.name}";
+            newTaskText.text = Resources.Load<TaskDesc>(newTaskObject.name).description;
         } 
     }
 
+    private void InitialiseStartList()
+    {
+		for (var i = 0; i < m_taskList.Count; i++)
+		{
+			foreach (var pair in m_taskList)
+			{
+				var newTaskObject = Instantiate(taskTextPrefab, taskContainer.transform);
+
+				newTaskObject.name = pair.Key.ToString();
+
+				var newTaskText = newTaskObject.GetComponent<TextMeshProUGUI>();
+				newTaskText.fontSize = fontSize * 2.5f;
+				newTaskText.text = Resources.Load<TaskDesc>(newTaskObject.name).description;
+			}
+		}
+	}
+
 /// <summary>
 /// Updates the progress value displayed on the clipboard for a given task.
 /// </summary>
-/// <param name="taskName">String identifier of the task to update.</param>
-/// <param name="taskProgress">Progress of the task to be displayed on the clipboard.</param>
-    public void UpdateClipboardTask(string scene)
+/// <param name="scene">String identifier of the task to update.</param>
+    public void UpdateClipboardTask(TaskType type)
     {
-        taskContainer.transform.Find(scene).GetComponent<TextMeshProUGUI>().text = $"<s>{scene}</s>";
+		Debug.Log(type.ToString());
+		taskContainer.transform.Find(type.ToString()).GetComponent<TextMeshProUGUI>().text = $"<s>{taskContainer.transform.Find(type.ToString()).GetComponent<TextMeshProUGUI>().text}</s>"; //this is the worst line of c# ever written.
```

Comment:
This is pre bad lol but it's an easy fix to make it look a little nicer. Also try to remove debug stuff like this once u know it works.

```suggestion
		var taskText = taskContainer.transform.Find(type.ToString()).GetComponent<TextMeshProUGUI>();
taskText.text = $"<s>{taskText.text}</s>";
```